### PR TITLE
Updating arns to include wildcard

### PR DESCRIPTION
**Pull request by sshaikRC**
### Submitter Checklist
-  [ ] Is there an impact on Auditing and Logging procedures or capabilities?
-  [ ] Is there an impact on Authentication procedures or capabilities?
-  [ ] Is there an impact on Authorization procedures or capabilities?
-  [ ] Is there an impact on Communication Security procedures or capabilities?
-  [ ] Is there an impact on Cryptography procedures or capabilities?
-  [ ] Is there an impact on Sensitive Data procedures or capabilities?
-  [ ] Is there an impact on any other security-related procedures or capabilities?
-  [ ] No security impacts identified.

Security Risks Identified - For any applicable items on the "Submitter Checklist," describe the impact of the change and any implemented mitigations.

### CODE REVIEW

**Comment by robo-gotham** [comment]
Snyk Scanning for Commit: 42f99c735bc5a1492052829a7339e810af8f990d

Snyk Infrastructure as Code

- Snyk testing Infrastructure as Code configuration issues.
✔ Test completed.

Issues
  No vulnerable paths were found!

-------------------------------------------------------

</details>

Test Summary

  Organization: batcave-ispg
  Project name: CMS-Enterprise/batcave-tf-eso

✔ Files without issues: 4
✗ Files with issues: 0
  Ignored issues: 0
  Total issues: 0 [ 0 critical, 0 high, 0 medium, 0 low ]

-------------------------------------------------------

Report Complete

  Your test results are available at: https://snyk.io/org/batcave-ispg/projects
  under the name: CMS-Enterprise/batcave-tf-eso

File: iam.tf [hunk 1]
```diff
@@ -25,6 +26,10 @@ locals {
         "batcave/sonar-registry-credentials",
         "batcave/sonar-agent-api-key",
         "batcave/kiali",
-        "batcave/gitlab-rails-secret-s3",
+        "batcave/gitlab-rails-secret-s3"
```
Comment (bushong1) [inline iam.tf]: Why did the trailing comma get stripped?

File: iam.tf [hunk 2]
```diff
@@ -15,7 +15,8 @@ module "external_secrets_irsa" {
 }
 
 locals {
-    secret_arns = [ for name in local.secret_names : "arn:aws:secretsmanager:${var.aws_region}:${var.aws_id}:secret:${name}" ]
+    secret_arns = [ for name in local.secret_names : "arn:aws:secretsmanager:${var.aws_region}:${var.aws_id}:secret:${name}*"
+    ]
```
Comment (bushong1) [inline iam.tf]: Why did this move the the line below?

Comment (sshaikRC) [reply iam.tf]: Fixed this

File: iam.tf (hunk 1 above)
Comment (sshaikRC) [reply iam.tf]: Added a few arns, there doesn't need to be a comma after the last entry in the list right?

File: iam.tf [hunk 3]
```diff
@@ -26,5 +26,13 @@ locals {
         "batcave/sonar-agent-api-key",
         "batcave/kiali",
         "batcave/gitlab-rails-secret-s3",
+        "batcave/alertmanager-secret",
+        "batcave/loki-write-keys",
+        "batcave/gitlab-secret",
+        "batcave/gitlab-rails-secret-backup"
```
Comment (bushong1) [inline iam.tf:32]: These gitlab keys will only be used in batcave clusters.  might want to make them conditionally added based on a far?

Comment (bushong1) [reply iam.tf:32]: Lets clean this up after release

**Review by bushong1 — APPROVED** [review]

**Pull request merged**